app/pages/notifications/notification-section.jsx

```diff
@@ -274,7 +276,7 @@ export default class NotificationSection extends Component {
               pageCount={this.state.lastMeta.page_count}
               pageSelector={false}
               previousLabel={<span><i className="fa fa-chevron-left" /> previous</span>}
-              totalItems={<span className="notification-section__item-count">{(l.page * l.page_size) - (l.page_size - 1)} - {Math.min(l.page_size * l.page, l.count)} of {l.count}</span>}
+              totalItems={<span className="notification-section__item-count">{firstNotification} - {lastNotification} of {l.count}</span>}
```

Comment:
Yes, passing in calculations threw a warning, but getting the value beforehand worked alright.